### PR TITLE
Needs to be installable with bundler.

### PR DESCRIPTION
**Pull request by JonRowe**
Test.

### CODE REVIEW

**Comment by halostatue** [comment]
I saw a successful bundle install, and the test failure is the expected test failure (the `ldiff` comparison between `hello` and `hella` returns a non-zero exit code).

**Comment by JonRowe** [comment]
Yep I'm now very confused, something else must have changed in the same nightly build, (we didnt change anything and the bundler output indicates diff-lcs is our failure cause)

**Comment by halostatue** [comment]
No, the bundler output indicates that `rubocop` is the failure cause. Somehow, even though you have indicated that you exclude Rubocop, it’s *not* being excluded the way that you think it is on Ruby 1.8.7.

**Comment by JonRowe** [comment]
Ok we'll play it your way, if rubocop is to blame this shouldn't work.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gem 'diff-lcs', '>= 1.3.0'
+
+# this won't install because Ruby 1.8.7 is not Ruby 1.9.x
+gem 'rubocop', '~> 0.32.1', platform: [:ruby_19]
```
Comment (halostatue) [inline Gemfile]: `:platform => [ :ruby_19 ]`

Comment (JonRowe) [inline Gemfile]: Yes yes yes, already fixed :P

Comment (JonRowe) [inline Gemfile]: Like (hopefully) most people I don't actually use 1.8.7 for the day job ;)

**Comment by JonRowe** [comment]
Ok, so it seems bundler is now ignoring platform directives? Fun. Thanks for the assist.

**Comment by halostatue** [comment]
Glad I could help.

**Pull request closed without merging**